tests/manage/mcg/test_multi_region.py

```diff
@@ -66,13 +65,17 @@ def test_multiregion_bucket_creation(self, mcg_obj, multiregion_mirror_setup):
     @bugzilla('1827317')
     @skipif_ocs_version("==4.4")
     @pytest.mark.polarion_id("OCS-1784")
-    def test_multiregion_mirror(self, mcg_obj, awscli_pod, multiregion_mirror_setup):
+    def test_multiregion_mirror(self, cld_mgr, mcg_obj, awscli_pod, multiregion_mirror_setup):
         """
         Test multi-region bucket creation using the S3 SDK
         """
 
-        bucket, backingstore1, backingstore2 = multiregion_mirror_setup
+        bucket, backingstores = multiregion_mirror_setup
+        backingstore1 = backingstores[0]
```

Comment:
any chance we will want to increase to more than 2? maybe better to unpack it in iterations and append to a list?

Reply:
Right now, the number is two just because we didn't want to change the logic of the test. The factory isn't limited in its amount of parameterization, and you can see that we specifically requested two backingstores [here](https://github.com/red-hat-storage/ocs-ci/pull/2210/files#diff-484462fced51d1a06b1d93b4a44dd535R2116).
Once the PR is merged, we can alter the existing test case, or add new ones that'll work with more than two backingstores :)